tldw_Server_API/app/api/v1/endpoints/flashcards.py

```
@@ -929,6 +930,20 @@ def list_flashcards(
         raise HTTPException(status_code=500, detail="Failed to list flashcards") from e
 
 
+@router.get("/tags", response_model=FlashcardTagSuggestionsResponse)
+def list_flashcard_tag_suggestions(
+    q: Optional[str] = None,
+    limit: int = Query(50, ge=1, le=1000),
+    db: CharactersRAGDB = Depends(get_chacha_db_for_user),
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)